uhabits-android/src/main/java/org/isoron/uhabits/receivers/ReminderReceiver.kt

```diff
@@ -96,7 +96,7 @@ class ReminderReceiver : BroadcastReceiver() {
                 }
                 Intent.ACTION_BOOT_COMPLETED -> {
                     Log.d("ReminderReceiver", "onBootCompleted")
-                    reminderController.onBootCompleted()
+                    // NOTE: Some activity is executed after boot through HabitsApplication, so receiving ACTION_BOOT_COMPLETED is essential.
                 }
```

Comment:
Could you clarify why do we still need this block? We would still receive `ACTION_BOOT_COMPLETED`, even if we remove the block.

Reply:
The block is not needed (and I can see the intent is also logged above); What I deemed important is that somewhere it is noted that it is important that the `ACTION_BOOT_COMPLETED` intent is received.

Actually, as for both `ACTION_BOOT_COMPLETED` and `MY_PACKAGE_REPLACED` the only thing we need is that the application is started (and all the code from `ReminderReceiver` is not needed), I would suggest that both should be received by a common dummy receiver (as currently with `UpdateReceiver`), which we could call `StartAppReceiver` or similar. Then it is clear that these intents are solely received to let the app start.